protected/humhub/modules/space/modules/manage/views/default/index.php

```diff
@@ -27,6 +27,7 @@
 
         <?= SpaceNameColorInput::widget(['form' => $form, 'model' => $model]) ?>
         <?= $form->field($model, 'description')->textarea(['rows' => 6]); ?>
+        <?= $form->field($model, 'long_description')->textarea(['rows' => 6]); ?>
```

Comment:
- Old description should not longer be a text area
- New description should be a Markdown Richtext Field (https://github.com/humhub/humhub/blob/master/protected/humhub/modules/content/widgets/richtext/RichTextField.php)